resources/views/user/post/scheduling.blade.php

```diff
@@ -267,4 +268,38 @@
             </div>
         </div>
     </div>
+
+    <div class="modal fade bd-example-modal-xl" id="postVerification" tabindex="-1" role="dialog" aria-labelledby="addAccountModalLabel" aria-hidden="true">
+        <div class="modal-dialog  modal-xl modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">Post Verification</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <div class="table-responsive">
+                        <table class="table table-borderless" data-toggle="table" data-pagination="true" data-search="true">
+                            <caption></caption>
+                            <thead>
+                                <tr>
+                                    <th scope="col" rowspan="2" data-sortable="true">#</th>
+                                    <th scope="col" rowspan="2" data-sortable="true" data-field="Platform">Platform</th>
+                                    <th scope="col" rowspan="2" data-sortable="true" data-field="supported_type">Supported Media Type</th>
+                                    <th scope="col" colspan="2">Max Size Supported</th>
+                                </tr>
+                                <tr>
+                                    <th scope="col" data-sortable="true" data-field="images_size">Image</th>
+                                    <th scope="col" data-sortable="true" data-field="video_size">Video</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                            </tbody>
```

Comment:
Why body is empty?

Reply:
Spandan Has to give that to me so initially he was going to do it now I'll do it on priority

Reply:
Okay

Reply:
Please Check Now I have pushed the changes